pkg/util/log/audit/audit.go

```diff
@@ -65,125 +72,146 @@ var (
 
 // NewEntry returns a log entry that embeds the provided logger. It has a hook
 // that knows how to hydrate an IFxAudit log payload before logging it.
-func NewEntry(env env.Core, logger *logrus.Logger) *logrus.Entry {
-	logger.AddHook(&payloadHook{
+func NewEntry(logger *logrus.Logger) *logrus.Entry {
+	// clone logger to avoid audit logs leaking into non-audit logs
+	auditLogger := logrus.New()
+	auditLogger.Out = logger.Out
+	auditLogger.Formatter = logger.Formatter
+	auditLogger.ReportCaller = logger.ReportCaller
+	auditLogger.Level = logger.Level
+	auditLogger.ExitFunc = logger.ExitFunc
+
+	// adding the payload hook first to ensure alll subsequent hooks received
+	// the mutated entry
+	auditLogger.AddHook(&payloadHook{
```

Comment:
What is the reasoning why we now initiate new object with each request instead of having it part of Frontend object?